File: src/main/java/com/back/nbe9112team06/domain/meeting/repository/MeetingRepository.java
package com.back.nbe9112team06.domain.meeting.repository;

import com.back.nbe9112team06.domain.meeting.entity.Meeting;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;
import java.util.Optional;

public interface MeetingRepository extends JpaRepository<Meeting, Integer> {
    boolean existsByRandomUrl(String randomUrl);
    Optional<Meeting> findByRandomUrl(String randomUrl);
    List<Meeting> findByMember_IdOrderByCreatedAtDesc(Integer memberId);
}

File: src/main/kotlin/com/back/nbe9112team06/domain/meeting/repository/MeetingRepository.kt
package com.back.nbe9112team06.domain.meeting.repository

import com.back.nbe9112team06.domain.meeting.entity.Meeting
import org.springframework.data.jpa.repository.JpaRepository

interface MeetingRepository : JpaRepository<Meeting, Int> {
    fun existsByRandomUrl(randomUrl: String): Boolean
    fun findByRandomUrl(randomUrl: String): Meeting?
    fun findByMember_IdOrderByCreatedAtDesc(memberId: Int): List<Meeting>
}

